openid-4-verifiable-presentations-1_0.md

```diff
@@ -740,6 +740,9 @@ in the mdoc, e.g., `first_name`.
 If the `values` property is present, the Wallet SHOULD return the claim only if the
 type and value of the claim both match for at least one of the elements in the array. Details of the processing rules are defined in (#selecting_claims).
 
+`intent_to_retain`
+: OPTIONAL. A boolean that indicates the Verifier intends to retain the Claim's data that is being requested.
```

Comment:
```suggestion
: OPTIONAL. A boolean that indicates the Verifier intends to store the Claim's data that is being requested beyond the processing of the response, for example, for purposes of recording a proof of the transaction or to maintain a user profile.
```
Not sure if this is the best wording, but we should try to explain what the 'retain' in `intent_to_retain` means.